zerver/migrations/0319_create_edit_message_array_and_backfill_edit_messages.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.1.7 on 2021-04-14 11:26
+import django.contrib.postgres.fields
+import orjson
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def backfill_edit_messages(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Message = apps.get_model("zerver", "Message")
+    ArchivedMessage = apps.get_model("zerver", "ArchivedMessage")
+    msg_models = [Message, ArchivedMessage]
+
+    for model in msg_models:
+        for message in model.objects.all():
```

Comment:
You'll definitely want this to filter for `edit_history` not being null -- it'll dramatically reduce the number of rows to process.

(Or maybe that's not possible, because we need to set the default to `[]`?)

Reply:
I think that is possible, the default value of the ArrayField is null. Will make the change!